core/src/main/resources/application.conf

```diff
@@ -118,6 +120,11 @@ call-caching {
   // request to an external service (DockerHub, GCR).  If a call fails to lookup a
   // Docker hash, it will fail.
   lookup-docker-hash = false
+
+  // This option determines whether file contents are hashed to determine cache hit or miss.
+  hash-file-contents = true
+  // This option determines whether file paths are hashed to determine cache hit or miss.
+  hash-file-paths = false
```

Comment:
If you ran once with both Docker hashes on, and then a second time with only the name hashing on, would you really expect a cache hit?  It seems we'd have to record aggregate hashes for the first run with and without the docker image hash included.